packages/components/src/Components/Battery/Batery.test.js

```diff
@@ -42,6 +42,7 @@ describe('Battery component', () => {
         it('NullBatery, default', () => {
             const wrapper = shallow(<Battery severity={''} label={''}/>);
             expect(toJson(wrapper)).toMatchSnapshot();
+            // eslint-disable-next-line no-console
```

Comment:
I think that we should permit warn and error but blacklist log.

We could use this rule globally: `no-console: ["error", { allow: ["warn", "error"] }]`

What do you think?

Reply:
I don't really like `error` for logs, but allowing warnings and errors is a good idea. I'll update it!